src/analysis/retail/deathknight/frost/CHANGELOG.tsx

```diff
@@ -5,19 +5,22 @@ import { Khazak, ToppleTheNun } from 'CONTRIBUTORS';
 import { SpellLink } from 'interface';
 
 export default [
+  change(date(2024, 9, 21), 'Updates for 11.0.2: General bug fixes and tweaks to ensure spec matches live - Boohbah', Khazak),
   change(date(2024, 4, 22), <>Rework <SpellLink spell={talents.KILLING_MACHINE_TALENT} /> guide to look at current Runes when determining waste</>, Khazak),
   change(date(2024, 4, 22), 'Bump compatibility to 10.2.6.', Khazak),
   change(date(2023, 7, 17), 'Bump compatibility to 10.1.5.', Khazak),
   change(date(2023, 6, 19), 'Update SpellLink usage.', ToppleTheNun),
   change(date(2023, 6, 18), <>Fixed some casts of <SpellLink spell={talents.BREATH_OF_SINDRAGOSA_TALENT} /> being improperly counted</>, Khazak),
-  change(date(2023, 4, 21), <>Added guide section for <SpellLink spell={talents.EMPOWER_RUNE_WEAPON_SHARED_TALENT} /></>, Khazak),
+  change(date(2023, 4, 21), <>Added guide section for EMPOWER_RUNE_WEAPON_SHARED_TALENT</>, Khazak),
+  //change(date(2023, 4, 21), <>Added guide section for <SpellLink spell={talents.EMPOWER_RUNE_WEAPON_SHARED_TALENT} /></>, Khazak),
   change(date(2023, 4, 11), 'Updated example report', Khazak),
   change(date(2023, 4, 6), <>Changed logic in <SpellLink spell={talents.KILLING_MACHINE_TALENT} /> module to show refreshes that happen during GCD lock</>, Khazak),
   change(date(2023, 4, 6), <>Improved <SpellLink spell={talents.HORN_OF_WINTER_TALENT} /> module to include RP gained while <SpellLink spell={spells.RUNE_OF_HYSTERIA_BUFF} /> is active</>, Khazak),
   change(date(2023, 4, 6), <>Improved <SpellLink spell={talents.KILLING_MACHINE_TALENT} /> module to account for extra refreshes when going from 2 to 1 stacks</>, Khazak),
   change(date(2023, 3, 31), <>Fixed uptime suggestions for <SpellLink spell={talents.BREATH_OF_SINDRAGOSA_TALENT} /></>, Khazak),
   change(date(2023, 3, 30), <>Added guide section for <SpellLink spell={talents.HORN_OF_WINTER_TALENT} /></>, Khazak),
-  change(date(2023, 3, 24), <>Improved <SpellLink spell={talents.RIME_TALENT} /> and <SpellLink spell={talents.KILLING_MACHINE_TALENT} /> stats</>, Khazak),
+  change(date(2023, 3, 24), <>Improved RIME_TALENT and <SpellLink spell={talents.KILLING_MACHINE_TALENT} /> stats</>, Khazak),
```

Comment:
Go ahead and clear the changelog and start it with a single entry for 11.0. We don't keep changelogs between xpacs to avoid bloating them without outdated info. It also avoids the issue you hit here with spell ids no longer working. Also, create an entry for yourself in the contributors file and pass that in for the author entry.